src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/TracerProviderBuilderExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// <copyright file="TracerProviderBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation;
+using OpenTelemetry.Resources;
+
+namespace OpenTelemetry.Trace
+{
+    /// <summary>
+    /// Extension class for TracerProviderBuilder.
+    /// </summary>
+    public static class TracerProviderBuilderExtensions
+    {
+        /// <summary>
+        /// Add AWS Lambda configurations.
+        /// </summary>
+        /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+        /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+        public static TracerProviderBuilder AddLambdaConfigurations(this TracerProviderBuilder builder)
```

Comment:
Better to call it `AddAWSLambdaInstrumentation()`? We are actually adding an instrumentation so let's just call it that. Using "configurations" might confuse users into thinking that there's actually a way to configure the lambda instrumentation in some way here.

Reply:
We don't actually add instrumentation in the `TracerProvider`, the instrumentation is manual in this PR and cx has to call API in this library to get the Lambda handler instrumented. What we add to `TracerProvider` is a resource detector and source.

Reply:
Hmm. I don't really know what we can better call this function. Typically calling something "configuration" means that customers are able to configure something here but that's not the case right now. Do we see some actual configuration options going into this method in the future? If not, we should rename it as `AddAWSLambdaProperties` or something. Also be sure to call it "AWS"Lambda than just Lambda to be more specific.

Reply:
Yes, I would foresee that we're going to add more config options in future, so we can keep the name. Also, will add the `AWS` to the name to specify.